src/service_main.rs

```diff
@@ -25,19 +25,31 @@ extern crate serde;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde_json;
+extern crate failure;
+#[macro_use(
+   slog_o, slog_info, slog_kv, slog_log, slog_record, slog_b, slog_record_static, slog_error
+)]
+extern crate slog;
+extern crate slog_async;
+extern crate slog_mozlog_json;
+extern crate slog_term;
+extern crate mozsvc_common;
 
 mod app_errors;
 mod auth_db;
 mod bounces;
 mod deserialize;
 mod duration;
+mod logging;
 mod providers;
 mod send;
 mod settings;
 mod validate;
 
 fn main() {
+    let logger = logging::init_logging(&settings::Settings::new().unwrap()).unwrap(); 
```

Comment:
Hmm, this smells a bit funny because we're instantiating `Settings` twice in this process now. Once here, and once in `src/send/mod.rs`. That is mostly my fault for using `lazy_static` rather than a sensible injection strategy (see #49).

What I think would be better is if we `manage(settings)` below like you already are for `logger`. Then the `send` module could just add it as a param, see the Rocket docs about managed state for more on this:

https://rocket.rs/guide/state/#managed-state

Reply:
Phil, I realized this logger line is not needed for now. That was leftover code from some experiments we were doing earlier.

Anyways, managed stated would be nice. I can work on that after I'm finished with this :)

Reply:
I agree with that. I think probably it would be best to send this in as another PR though, fixing specifically the #49 bug. I'll work on that.